File: API/DTOs/Pagination/AllUsersParams.cs
namespace API.DTOs.Pagination
{
    public class AllUsersParams : BaseParams
    {
        private string _scvRoles;
        public string CsvRoles
        {
            get => _scvRoles;
            set => _scvRoles = string.IsNullOrEmpty(value) ? "" : value.ToLower();
        }

        private string _lockUnlock;
        public string LockUnlock
        {
            get => _lockUnlock;
            set => _lockUnlock = string.IsNullOrEmpty(value) ? "" : value.ToLower();
        }

        private string _activation;
        public string Activation
        {
            get => _activation;
            set => _activation = string.IsNullOrEmpty(value) ? "" : value.ToLower();
        }
    }
}

File: ClientApp/src/app/shared/models/pagination/allUsersParams.ts
import { BaseParams } from "./baseParams";

export class AllUsersParams extends BaseParams {
    csvRoles: string | undefined;
    lockUnlock: string | undefined;
    activation: string | undefined;

    constructor(pageSize: number = 10, pageNumber: number = 1, sortBy: string = '', searchBy: string = '',
        csvRoles: string | undefined = undefined,
        lockUnlock: string | undefined = undefined,
        activation: string | undefined = undefined) {
        super(pageSize, pageNumber, sortBy, searchBy);

        this.csvRoles = csvRoles;
        this.lockUnlock = lockUnlock;
        this.activation = activation;
    }
}